### ISSUE TITLE
Feature request: Ability to "uninvolve" myself

### ISSUE THREAD

**Issue by nate-tri** [Reviewable/Reviewable#599]
For reviews that someone has FYI'd me, or mentioned me once in some sidebar, it would be good to be able to uninvolve myself once I believe my part is complete. Perhaps some sort of review-level withdraw?

This would allow me to remove reviews from my dashboard that are no longer relevant to me and allow me to focus only on the reviews that I am actually going to be involved in.

**Comment by pkaminski** [comment]
I believe this is a bug: the main review page knows that mentionees are different from reviewers, but the dashboard treats them the same for some reason.  I'll see if I can fix it.

**Comment by pkaminski** [comment]
Should be fixed now, let me know if it did/didn't help.
